.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,80 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
+            - develop
+            - '[0-9]+.[0-9]+'
+
+jobs:
+    test:
+        name: "Run tests with php ${{ matrix.php-version }}"
+        runs-on: ubuntu-18.04
```

Comment:
why not `ubuntu-latest`?

Reply:
Wasn't sure if we get any problems if they switch to ubuntu 20.04

Reply:
If I'm not wrong we used `ubuntu-latest` on the other repos too